README.md

```diff
@@ -1 +1,1901 @@
-# react-blog
\ No newline at end of file
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Probably a good idea to replace this boilerplate file with one that actually provides some useful instructions to someone who will use this software to create a blog